language/transaction-builder-generator/src/rust.rs

```diff
@@ -159,3 +162,49 @@ fn make_transaction_argument(type_tag: &TypeTag, name: &str) -> String {
         Struct(_) | Signer => type_not_allowed(type_tag),
     }
 }
+
+pub struct Installer {
+    install_dir: PathBuf,
+    libra_types_version: String,
+}
+
+impl Installer {
+    pub fn new(install_dir: PathBuf, libra_types_version: String) -> Self {
+        Installer {
+            install_dir,
+            libra_types_version,
+        }
+    }
+}
+
+impl crate::SourceInstaller for Installer {
+    type Error = Box<dyn std::error::Error>;
+
+    fn install_transaction_builders(
+        &self,
+        name: &str,
+        abis: &[ScriptABI],
+    ) -> std::result::Result<(), Self::Error> {
+        let dir_path = self.install_dir.join(name);
+        std::fs::create_dir_all(&dir_path)?;
+        let mut cargo = std::fs::File::create(&dir_path.join("Cargo.toml"))?;
+        write!(
+            cargo,
+            r#"[package]
+name = "{}"
+version = "0.1.0"
+edition = "2018"
+
+[dependencies]
+serde = {{ version = "1.0", features = ["derive"] }}
```

Comment:
Not at all crucial for this PR, but might be worth thinking about eventually: If there's a good way to handle dependency bumps for external dependencies without having to manually change the generated code here.

Reply:
Yes, that's a good point. I need to track this somewhere.